config/cucumber.yml

```diff
@@ -1,10 +1,9 @@
 <% std_opts = "--strict --tags ~@wip #{ENV['CUCUMBER_OPTIONS']}" %>
 default: <%= std_opts %>
-parallel: --backtrace --format junit --out tmp/junit/cucumber-<%= [ENV['MULTIJOB_KIND'], Process.pid].compact.join('-') %>  --format ParallelTests::Cucumber::FailuresLogger --out tmp/junit/cucumber_<%= ENV['MULTIJOB_KIND'] %>_failures.log --strict --tags ~@wip -r features --format progress
```

Comment:
Lets make it cleaner, remove all MULTIJOB_KIND

```bash
~/Devel/3scale/porta (master)$ rg MULTIJOB_KIND                                                                                                                                                                            [2.4]
test/test_helper.rb
25:junit = MiniTest::Reporters::JUnitReporter.new("tmp/junit/unit-#{[ENV['MULTIJOB_KIND'], Process.pid].compact.join('-')}")

config/cucumber.yml
3:parallel: --backtrace --format junit --out tmp/junit/cucumber-<%= [ENV['MULTIJOB_KIND'], Process.pid].compact.join('-') %>  --format ParallelTests::Cucumber::FailuresLogger --out tmp/junit/cucumber_<%= ENV['MULTIJOB_KIND'] %>_failures.log --strict --tags ~@wip -r features --format progress

spec/rails_helper.rb
121:    junit = "tmp/junit/spec-#{[ENV['MULTIJOB_KIND'], Process.pid].compact.join('-')}/spec.xml"
```

Reply:
Awesome! 👍 🥇 💪 💃

Reply:
I do not know if circleci use the JUnit reports to display the failed tests 🤔